docs/usage/TYPE_SYNC.md

```diff
@@ -0,0 +1,120 @@
+# commercetools type sync
+
+Utility which provides API for building CTP type update actions and type synchronisation.
+
```

Comment:
provides a API

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/f4fc41525cbd101ff25de148431849084815c98a#diff-649a74fcd11f3ef15302f4486a9ddbd1R3

Reply:
I would also appreciate it if you fix it in the similar docs `docs/usage/CATEGORY_SYNC.md`, `docs/usage/PRODUCT_SYNC.md`, etc.. Since the exact same sentence exists there also (but you don't have to in this PR 😉 )